src/main/java/com/w6/data/dao/event/EventServiceImpl.java

```diff
@@ -42,7 +42,48 @@ public Event save(Event event) {
     }
 
     @Override
-    public long count() {
-        return eventRepository.count();
+    public List<Event> findAll() {
+        List<Event> result = new ArrayList<>();
+
+        Iterable<Event> events = eventRepository.findAll();
```

Comment:
In the link above there is no information how to get `Stream` from `Iterable` since `Iterable` doesn't have `stream()` method. But you can do it on your own in one line.
Done.